internal/npm_package/packager.js

```diff
@@ -86,6 +86,9 @@ function main(args) {
 
   // src like baseDir/my/path is just copied to outDir/my/path
   for (src of srcsArg.split(',').filter(s => !!s)) {
+    if (!src.startsWith(baseDir)) {
+      throw new Error(`Files in srcs must reside in base directory: ${src}`);
```

Comment:
can we be more helpful? "looks like you meant to put this label in `deps` instead"